pvacseq/lib/fasta_generator.py

```diff
@@ -157,3 +157,55 @@ def execute(self):
         reader.close()
         writer.close()
         key_writer.close()
+
+class FusionFastaGenerator(FastaGenerator):
+    def execute(self):
+        peptide_sequence_length = self.peptide_sequence_length
+        reader                  = open(self.input_file, 'r')
+        tsvin                   = csv.DictReader(reader, delimiter='\t')
+        fasta_sequences         = OrderedDict()
+        for line in tsvin:
+            variant_type = line['variant_type']
+            position     = int(line['fusion_position'])
+            sequence     = line['fusion_amino_acid_sequence']
+            one_flanking_sequence_length = self.determine_flanking_sequence_length(len(sequence), peptide_sequence_length, line)
+            if position < one_flanking_sequence_length:
+                start_position = 0
+            else:
+                start_position = int(position - one_flanking_sequence_length)
+
+            if variant_type == 'inframe_fusion':
+                stop_position = int(position + one_flanking_sequence_length)
+                subsequence   = sequence[start_position:stop_position]
+            elif variant_type == 'frameshift_fusion':
```

Comment:
What if the input value is neither inframe_fusion or frameshift_fusion?  Other people's files are messy - it should probably do something sensible and exit, right?